### ISSUE TITLE
ImageRequest function is Volley library is deprecated!

### ISSUE THREAD

**Issue by CloudyPadmal** [scorelab/kute#28]
**Issue**

The image request call made to fetch images from URL is deprecated. Picasso is a library specialized in image fetching compared to Volley. We can integrate that library instead of Volley as volley hasn't been use anywhere in the app except for image fetching

**Steps to reproduce**

1. Go to RegisterActivity class
2. Find "ImageRequest"

**Would you like to work on this issue?**

Yes and I can send a PR for review!

**Comment by icancode23** [comment]
@CloudyPadmal brother you can use image loader as well...using the same volley library..

**Comment by CloudyPadmal** [comment]
@icancode23 agreed! But picasso is simple to integrate than volley! Don't you agree? :)

**Comment by icancode23** [comment]
@CloudyPadmal no i was just saying that because then you won't need an additional dependency of picasso

**Comment by CloudyPadmal** [comment]
@icancode23 volley is only used in this image fetching. So instead of that we can use Picasso, a library specialized in image fetching from URLs to simplify the code!!

**Comment by icancode23** [comment]
@CloudyPadmal two developers..two different thinking....

**Comment by CloudyPadmal** [comment]
That's good for app development! :)

**Comment by vidu171** [comment]
Can I try to work on this. I have not worked with Picasso yet. But I think I can learn it and try to implement it.
